packages/core/src/hooks/useAddressBook.tsx

```diff
@@ -34,45 +41,65 @@ export default function useAddressBook(): [
 
   function getNewContactId(): number {
     if (addressBook.length === 0 || addressBook === undefined) return 1;
-    return Math.max(...addressBook.map((o) => o.contactid)) + 1;
+    return Math.max(...addressBook.map((o) => o.contactId)) + 1;
   }
```

Comment:
This should be rewritten as a useCallback hook. Something like:

```  const getNewContactId = useCallback(() => {
      if (addressBook.length === 0 || addressBook === undefined) return 1;
        return Math.max(...addressBook.map((o) => o.contactId)) + 1;
  }, [addressBook]);```

Reply:
Same comment applies to addContact/removeContact/getContactById/editContact -- they should be rewritten using useCallback